builder/FindPackages.cmake

```diff
@@ -144,8 +144,13 @@ function(configure_target target cflags libdirs)
   endforeach()
 endfunction()
 
-function(configure_libmfx_target target)
-  configure_target(${ARGV0} "${LIBMFX_CFLAGS}" "${LIBMFX_LIBRARY_DIRS}")
+function(configure_libmfx_target atarget)
+    # prefer local Dispatcher sources to preinstalled
+  if (TARGET mfx)
+     configure_target(${ARGV0} "${LIBMFX_CFLAGS}" "${LIBMFX_LIBRARY_DIRS}")
```

Comment:
I don't see that we setup these variables anywhere:
```
[dvrogozh@dvrscl MediaSDK]$ fgrep -rsn LIBMFX_CFLAGS
builder/FindPackages.cmake:148:  configure_target(${ARGV0} "${LIBMFX_CFLAGS}" "${LIBMFX_LIBRARY_DIRS}")
[dvrogozh@dvrscl MediaSDK]$ fgrep -rsn LIBMFX_LIBRARY_DIRS
builder/FindPackages.cmake:148:  configure_target(${ARGV0} "${LIBMFX_CFLAGS}" "${LIBMFX_LIBRARY_DIRS}")
[dvrogozh@dvrscl MediaSDK]$
```

Should not the code be just
if (NOT TARGET mfx)
  configure_target(${ARGV0} "${PKG_MFX_CFLAGS}" "${PKG_MFX_LIBRARY_DIRS}")
endif()

Reply:
I was sure that add_library(mfx) sets them. But in practice - looks they always empty. So ok.

Reply:
I am not sure I understood: you will drop this?